Introduce metadata about SIGs and markdown generation (#2648)
open-telemetry/semantic-conventions: SIGs.md

# Special Interest Groups (SIGs)

Semantic conventions in OpenTelemetry are organized into distinct areas,
with each area being overseen by a Special Interest Group (SIG). A SIG is
composed of multidisciplinary contributors who bring expertise in both the
relevant domain and OpenTelemetry itself.

SIGs have several responsibilities and goals, but their primary focus is to
define and guide semantic conventions toward stabilization. They are also tasked
with the long-term maintenance of these conventions, serving as domain experts
for pull request (PR) reviews and issue triage.

This document provides an overview of all SIGs involved in the
semantic conventions project. In particular, the table below highlights
the owners of each SIG and their current status.

> To learn more about SIGs for example, how to kick-start a new one, see the

Reviewer: wonder if we can 
- add a warning that this table is under development
- merge this PR 
- divide and conquer filling SIG details

WDYT?

# Special Interest Groups (SIGs)

Semantic conventions in OpenTelemetry are organized into distinct areas,

> with each area being overseen by a Special Interest Group (SIG). A SIG is

Reviewer: ```suggestion
with some of them being overseen by Special Interest Group (SIG).
```

we actually have full coverage, right?

composed of multidisciplinary contributors who bring expertise in both the
relevant domain and OpenTelemetry itself.

SIGs have several responsibilities and goals, but their primary focus is to

> define and guide semantic conventions toward stabilization. They are also tasked
> with the long-term maintenance of these conventions, serving as domain experts
> for pull request (PR) reviews and issue triage.

Reviewer: maybe something around these lines?
There is a push in the commuity repo to have smaller, well-scoped projects and permanent SIGs are seen more as an exception than a common scenario. E.g. we had HTTP or JVM  SIG but they had achieve their goals and are dissolved now.

```suggestion
define and guide semantic conventions toward stabilization. SIGs that achieve their goal 
may be dissolved, but the codeowner group stays serving as domain experts
for pull request (PR) reviews and issue triage.
```

> This document provides an overview of all SIGs involved in the

Reviewer: ```suggestion
This document provides an overview of all SIGs and codeowner groups involved in the
```
